### PR TITLE
Reorganize method classes and functions throughout entire pysyft project

### PR DESCRIPTION
**Pull request by tblazina**
There was previously no logic for how class methods were organized.
This will organize class methods with the following logic:

class SomeClass(object):
    def __magic_methods__(self):
        "magic methods first, usually in alphabetical order"
    def _private_method(self):
        "worker methods next, also in alphabetical order"
    def a_method(self):
        "then normal methods, also in alphabetical order"

### CODE REVIEW

**Comment by tblazina** [comment]
This closes #350

**Comment by tblazina** [comment]
good catch! I've moved it in. thanks a lot.

**Comment by tblazina** [comment]
@iamtrask  sure thing. I'll try to fix them this evening!

File: syft/he/paillier/basic.py [hunk 1]
```diff
@@ -119,17 +130,85 @@ def sum(self, dim=None):
             op = self.data.sum(axis=dim)
             return PaillierTensor(self.public_key, op, False, fixed_point_conf=self.fixed_point_conf)
 
+
+class Float():
```
Comment (bharathgs) [inline syft/he/paillier/basic.py]: Can you check this, again? this seems like an extra, I don't see it [here](https://github.com/OpenMined/PySyft/blob/master/syft/he/paillier/basic.py)

Comment (tblazina) [reply syft/he/paillier/basic.py]: hmm weird, I thought I had just pulled down from master. Let me take that out.

**Pull request merged**